docs/content/documentation/getting-started/configuration.md

```diff
@@ -135,6 +135,9 @@ smart_punctuation = false
 # For example, `![xx](...)` is ok but `![*x*x](...)` isn’t ok
 lazy_async_image = false
 
+# Whether footnotes are rendered in the GitHub-style (at the bottom, with back references) or plain.
+bottom_footnotes = false
```

Comment:
This bit of documentation (or the flag name?) was slightly ambiguous before using it. What is it meant by _plain_ here?

IIUC whether this flag is true or false, footnotes are _always_ rendered at the bottom, right? The only differences I notice is that the format (e.g. text[1] or text<sup>1</sup>) and if there a backlink or not.

In both cases it seems to work just great. Thanks @totikom !

Reply:
Have you tried reordering footnotes and text?
```markdown
[^2]: this footnote will appear the 2nd

Some text[^1] referencing[^2] footnotes[^3].

Another text mentioning footnotes[^2].

[^3]: this footnote should appear 3th
[^1]: this footnote should appear 1st
```

Reply:
oh interesting, I didn't try that! So, _plain_ here means "render footnotes where they are in source Markdown file". Maybe this could be worded a bit more explicit ion the comment. Or maybe it's just me, in this case ignore this comment :slightly_smiling_face:

Reply:
That's how it renders with my feature enabled:
![image](https://github.com/getzola/zola/assets/21067774/5a3e928e-83b3-48ac-88e8-e0294e536f29)
And that's how it renders with my feature disabled:
![image](https://github.com/getzola/zola/assets/21067774/414121b2-a6cb-4fa9-b7db-9addf9d79690)

Reply:
BTW, paragraphs in footnotes looks like an issue, but it is out of scope for this PR.